sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -13,19 +13,22 @@ public partial class CreateMessageBatchOptions
     }
     public sealed partial class ProcessErrorEventArgs : System.EventArgs
     {
+        public ProcessErrorEventArgs(System.Exception exception, Azure.Messaging.ServiceBus.ServiceBusErrorSource errorSource, string fullyQualifiedNamespace, string entityPath, string identifier, System.Threading.CancellationToken cancellationToken) { }
         public ProcessErrorEventArgs(System.Exception exception, Azure.Messaging.ServiceBus.ServiceBusErrorSource errorSource, string fullyQualifiedNamespace, string entityPath, System.Threading.CancellationToken cancellationToken) { }
         public System.Threading.CancellationToken CancellationToken { get { throw null; } }
         public string EntityPath { get { throw null; } }
         public Azure.Messaging.ServiceBus.ServiceBusErrorSource ErrorSource { get { throw null; } }
         public System.Exception Exception { get { throw null; } }
         public string FullyQualifiedNamespace { get { throw null; } }
+        public string Identifier { get { throw null; } }
```

Comment:
I got a comment from JS architect about whether `clientId` is better as `clientId` is used in most REST APIs.  Any thoughts? @JoshLove-msft @jsquire

Reply:
I'd prefer Identifier because we intentionally didn't call most of our types clients, only the ServiceBusClient has client in its name.

Reply:
Got! It looks that I have more work to do. I didn't read the description carefully and thought that we only had one identifier on the client that is passing down to all sub types

Reply:
@jeremymeng: There's an established pattern in Event Hubs using `Identifier`; we should look to keep Service Bus and Event Hubs aligned to the same pattern within each language.

I don't see any identifier-like members implemented for Event Hubs JS, so I have no objections to them being called `clientId` there, so long as Brian is comfortable with the drift between languages.

Reply:
@jsquire - The consideration for not using "client" in the names of the various Service Bus types apart from ServiceBusClient is relevant for all languages, not just .NET.